src/mesido/workflows/utils/error_types.py

```diff
@@ -0,0 +1,63 @@
+from enum import Enum
+
+from mesido.potential_errors import MesidoAssetIssueType, get_potential_errors
+
+# Convert potentials errors to errors for the following networks/definitions
+HEAT_NETWORK_ERRORS = "heat_network"
+HEAT_AND_COOL_NETWORK_ERRORS = "heat_and_cool_network"
+CUSTOM_ERRORS = "custom_errors"  # an example of custom stuff that can be added in the future
+
+
+def mesido_issue_type_gen_message(issue_type: MesidoAssetIssueType) -> str:
+    """
+    Get general message per issue type.
+
+    Returns
+    -------
+    String message
+    """
+    type_and_general_meassage = {
+        MesidoAssetIssueType.HEAT_DEMAND_POWER: "Asset insufficient installed capacity: please"
+        " increase the installed power or reduce the demand profile peak value of the demand(s)"
+        " listed.",
+        MesidoAssetIssueType.COLD_DEMAND_POWER: "Asset insufficient installed capacity: please"
+        " increase the installed power or reduce the demand profile peak value of the demand(s)"
+        " listed.",
+        MesidoAssetIssueType.HEAT_DEMAND_TYPE: "Incorrect asset type: please update.",
+        MesidoAssetIssueType.ASSET_PROFILE_CAPABILITY: "Profile assigment not allowed.",
+    }
+
+    return type_and_general_meassage[issue_type]
+
+
+def potential_error_to_error(network_check_type: Enum) -> None:
+    """
+    Convert potential errors to errors for the define error types
+
+    """
+
+    errors_on_types = {
+        HEAT_NETWORK_ERRORS: [
+            MesidoAssetIssueType.HEAT_DEMAND_POWER,
+            MesidoAssetIssueType.COLD_DEMAND_POWER,
+            MesidoAssetIssueType.HEAT_DEMAND_TYPE,
+            MesidoAssetIssueType.ASSET_PROFILE_CAPABILITY,
+        ],
+        HEAT_AND_COOL_NETWORK_ERRORS: [
+            MesidoAssetIssueType.HEAT_DEMAND_POWER,
+            MesidoAssetIssueType.COLD_DEMAND_POWER,
```

Comment:
Why are the ASSET_PROFILE_CAPABILITY and HEAT_DEMAND_TYPE not in the errors for heat and cooling networks? I think they also make sense for cooling

Reply:
Good catch, thanks. My mistake. The code has been updated